app/src/app/components/specmath-modal/modal.component.ts

```diff
@@ -30,19 +30,12 @@ export class ModalComponent implements OnInit {
   maxSteps: number;
   specNameInputOptions?: SpecNameInputOptions;
```

Comment:
Could we set defaults for these so that we don't have to consider the null case?